atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -550,6 +548,11 @@ private void sanityCheckRingConsistency() {
                 "Cassandra does not appear to have a consistent ring across all of its nodes. This could cause us to"
                         + " lose writes. The mapping of token ranges to hosts is:\n{}",
                 UnsafeArg.of("tokenRangesToHost", CassandraLogHelper.tokenRangesToServer(tokenRangesToHost)),
+                SafeArg.of(
```

Comment:
this seems like it'll be very big - will that cause memory issues?

Reply:
_Maybe_ but at this point we are in an inconsistent state and should not be writing to Cass anyway